fr-404.html

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Unfortunately github doesn't let us translate this file ;-(. It would break the site.

Reply:
Too bad! :(